pkg/api/postage_test.go

```diff
@@ -405,6 +410,9 @@ func TestChainState(t *testing.T) {
 		ts, _, _, _ := newTestServer(t, testServerOptions{
```

Comment:
this is a strange test case :) not sure what it is for...

Reply:
should i do anything with it?

Reply:
please remove it